.gitignore

```diff
@@ -11,6 +11,7 @@ __diff_output__
 /coverage
 /docs/.next
 /docs/export
+/docs/pages/playground.js
```

Comment:
I don't think that it would woof .gitignore prevent untracked files from being added. This file is already tracked by git and we want it to be tracked so we can quickly get started. 

```suggestion
```